kong/plugins/rate-limiting/migrations/003_10_to_112.lua

```diff
@@ -0,0 +1,12 @@
+return {
+  postgres = {
+    up = [[
+      CREATE INDEX IF NOT EXISTS ratelimiting_metrics_idx ON ratelimiting_metrics (service_id, route_id, period_date, period);
```

Comment:
Ugh, was about to comment here... Should we attempt to build this index concurrently? Especially considering the dataset size of the ratelimiting_metrics table. https://www.postgresql.org/docs/9.1/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY